knowledge-base/actions/sibz/github-status-action/action-security.yml

```diff
@@ -0,0 +1,11 @@
+name: 'github-status-action'
+github-token:
+  action-input:
+    input: authToken
+    is-default: false
+  permissions:
```

Comment:
I think the permission here should be `statuses: write`, to create status update for commits
view [here](https://github.com/Sibz/github-status-action/blob/650dd1a882a76dbbbc4576fb5974b8d22f29847f/src/main.ts#L48), [here](https://docs.github.com/en/rest/overview/permissions-required-for-github-apps#permission-on-statuses)